x-pack/plugins/observability/public/components/alerts_table/get_alerts_table_configuration.tsx

```diff
@@ -54,4 +54,5 @@ export const getAlertsTableConfiguration = (
     const { header, body, footer } = useGetAlertFlyoutComponents(observabilityRuleTypeRegistry);
     return { header, body, footer };
   },
+  ruleTypeIds: observabilityRuleTypeRegistry.list(),
```

Comment:
What does `ruleTypeIds` do? Why is it needed?

Reply:
> So like that in case when user associated only o11y alerts to a case we will represent the alert like in observability. And if the alert is only from ml then we will use the ml configuration.

So, you mean, we use this list to check whether alert (the related rule) belongs to observability or not, then, for example, the case will be created for that solution?
Does it also have any impact on, for example, the alert table?